### ISSUE TITLE
Force Maven Devfiles to use Maven

### ISSUE THREAD

**Issue by tsmaeder** [eclipse-che/che#16066]
Right now, our Maven devfile uses an example that also has a build.gradle. This will, in fact not be imported as a Maven project in jdt.ls, but as a Gradle project (Gradle has precedence).
There is preference in VS Code Java (something like "gradle.importer.enabled") that we can turn to false in the devfile in order to force use of Maven.

**Comment by benoitf** [comment]
IMHO issue is that both maven and gradle examples are using the same github project.
so maybe we could have two projects/repositories: one with pom.xml and the other one with build.gradle

To me it's strange to turn false something while for a maven example I expect to not see any gradle stuff in the project tree.

**Comment by tsmaeder** [comment]
@benoitf there is a separate issue for "use maven project for maven devfile". But it would still make sense to force use of maven with a maven stack.

**Comment by benoitf** [comment]
@tsmaeder yes makes sense on general purpose but the description of the issue is specifically about the example used in maven stack and then for the specific maven example it should not include any gradle file.

**Comment by l0rd** [comment]
I agree with @benoitf that to solve the specific issue (maven sample) we should split the sample in 2: one dedicated to maven and the other to gradle. And if there is already an issue about that ([this one](https://github.com/eclipse/che/issues/16051)?) I don't understand what this issues is about ("force use of maven with maven stack"?)

**Comment by tsmaeder** [comment]
If we have a project that has both maven and gradle configs (which is quite common), jdt.ls will pick gradle. This might not be the right thing to do. If we have a stack that is made for maven, we should always force usage of maven in the tooling (because that's what will be used in the dev container).

**Comment by l0rd** [comment]
@tsmaeder ok so your proposal is to update [the java-maven devfile in the registry](https://github.com/eclipse/che-devfile-registry/blob/master/devfiles/java-maven/devfile.yaml) to include the jdt ls option `gradle.importer.enabled=false`. I am ok with that as well and that would solve the original issue of https://github.com/eclipse/che/issues/16051.

**Comment by tsmaeder** [comment]
Well, for small values of "fix". Since gradle support is totally broken for new workspaces right now. We'll get a new release of VS Code Java later today where the issue is fixed and should update to that.

**Comment by l0rd** [comment]
@tsmaeder yes but the original issue is about using maven not gradle. And the difference is important since it's part of the happy path test that's used in all our PR checks.
